orcid_hub/views.py

```diff
@@ -1890,12 +1895,18 @@ def user_orgs_org(user_id, org_id=None):
     else:
         org = Organisation.get(id=org_id)
         uo, created = UserOrg.get_or_create(user_id=user_id, org_id=org_id)
-        if "is_admin" in data and uo.is_admin != data["is_admin"]:
+        if "is_admin" in data:
             uo.is_admin = data["is_admin"]
             uo.save()
         if "is_tech_contact" in data:
             user = User.get(id=user_id)
             if data["is_tech_contact"]:
+                # Updating old Technical Contact's Role info.
+                if org.tech_contact and org.tech_contact != user:
```

Comment:
DITO. 
also it looks like a perfect candidate for factoring it out just the way I suggested:

```
class Organisation:
    set_tech_contact(self, user):
        ...
```